### ISSUE TITLE
[Ironbreaker] Shield Sweep gives 10 grudges only once

### ISSUE THREAD

**Issue by SilverWF** [WarEmu/WarBugs#15634]
Shield Sweep must give 10 grudges **per each** target hit, but it gives 10 grudges per skill use, no matter how many targets got hit.

**Expected behavior and actual behavior:**
Expected: getting 10 grudges per each target hit
Actual: getting only 10 grudges per skill use

**Steps to reproduce the problem:**
Put around several targets
Get 25+ grudges
Use skill Shield Sweep
Look at new grudges and calculate

**Testing Screenshots/Videos/Evidences (always needed):**
https://www.youtube.com/watch?v=6OVGQ8EIg10
Skill cost is 25 grudges
We are getting 25 grudges, then pressing skill and hitting 3 targets.
We must have 35 grudges after this: 10*3 + 5 (from Rising Anger)
But we have only 15 grudges

**Comment by Balinorr** [comment]
Yeah, i've noted this too. Ability's grudge accumulation is still not working properly

**Comment by grunbag** [comment]
fixed with next patch
